packages/hint-compat-api/tests/css-next.ts

```diff
@@ -4,6 +4,8 @@ import readFile from 'hint/dist/src/lib/utils/fs/read-file';
 import { HintTest } from '@hint/utils-tests-helpers/dist/src/hint-test-type';
 import * as hintRunner from '@hint/utils-tests-helpers/dist/src/hint-runner';
 
+import { ignoredConnectors } from './fixtures/ignored-connectors';
```

Comment:
Is `fixtures` the best place to have this script?

Reply:
I tried with utils and a a complained about not having tests in that folder. I'll take a look see if there's another convention to use.

Reply:
From https://github.com/avajs/ava/blob/f4b2d196e813cf3549e19703e5a79ebc988fb8ca/docs/05-command-line.md

> Directories are recursed, with all *.js files being treated as test files. Directories named `fixtures`, `helpers` and `node_modules` are always ignored. So are files starting with `_` which allows you to place helpers in the same directory as your test files.

Reply:
I've renamed the file to `_ignored-connectors.ts` and moved it with the tests.